routers/router.go

```diff
@@ -0,0 +1,20 @@
+// @APIVersion 1.0.0
+// @Title WeatherAPI
+// @Weather API
+// @Contact karina.jimenez@globant.com
+package routers
+
+import (
+	"API-weather/controllers"
```

Comment:
It should be replace with `"github.com/kaiijimenez/API/controllers"`

Reply:
Done, curious though if it will work once you clone it in your machine